src/inmanta/agent/executor.py

```diff
@@ -91,6 +91,10 @@ class EnvBlueprint:
     requirements: Sequence[str]
```

Comment:
It feels a bit off to me that we still require users to pass a `Sequence` rather than a `Set` or `Collection`. This would require a different input type vs output/internal type, but afaik `dataclasses` does not support that. So there's probably nothing to be done about it (apart from dropping dataclass but that may be worse).

Reply:
I'm not super happy with it either and the fact we can't make it frozen and use `__post_init__` is unfortunate as well.